README.md

```diff
@@ -21,7 +21,7 @@ Tsuquyomi does not provide syntax-highlight nor indentation. You can use the fol
 * [leafgarland/typescript-vim](https://github.com/leafgarland/typescript-vim) provides syntax highlight.
 * [Quramy/vim-js-pretty-template](https://github.com/Quramy/vim-js-pretty-template) provides syntax highlight for contents in Template Strings.
 * [jason0x43/vim-js-indent](https://github.com/jason0x43/vim-js-indent) provides function of indent for both JavaScript and TypeScript.
-* [Quramy/vim-dtsm](https://github.com/Quramy/vim-dtsm) provides `.d.ts` management for [dtsm](with [https://github.com/vvakame/dtsm) users.
+* [Quramy/vim-dtsm](https://github.com/Quramy/vim-dtsm) provides `.d.ts` management for (with [dtsm](https://github.com/vvakame/dtsm) users).
```

Comment:
This will fix the invalid link, but the rendered sentence doesn't make sense: "d.ts management for (with dtsm users)." It should be:

    * [Quramy/vim-dtsm](https://github.com/Quramy/vim-dtsm) provides `.d.ts` management for [dtsm](https://github.com/vvakame/dtsm) users.